src/components/avatar/avatar.stories.mdx

```diff
@@ -35,7 +35,7 @@ const avatarStory = (args) => {
 
 Displays a user, organization or product [avatar](<https://en.wikipedia.org/wiki/Avatar_(computing)>) or profile picture.
 
-The size is fixed to the value of [our `sizes.$avatar-medium` token by default](/?path=/docs/design-tokens-sizes--page#sizing), making this an ideal pattern to use within [a Media object](/?path=/docs/objects-media--image). If the pattern contains no image, it will show a plain color.
+The size is fixed to the value of [our `sizes.$avatar-medium` token by default](/docs/design-tokens-sizes--page#sizing), making this an ideal pattern to use within [a Media object](/docs/objects-media--image). If the pattern contains no image, it will show a plain color.
```

Comment:
This first link is still broken for me 🤔 

I think it should maybe be this?

```suggestion
The size is fixed to the value of [our `sizes.$avatar-medium` token by default](/docs/design-tokens-size--page#sizing), making this an ideal pattern to use within [a Media object](/docs/objects-media--image). If the pattern contains no image, it will show a plain color.
```